00CREDITS

```diff
@@ -569,6 +569,7 @@ provided test systems where I was able to do development work.
 	Sergei Trofimovich (@trofi)
 	Danilo Spinella (@danyspin97) and
 	Jiajie Chen (@jiegec)
+	Jacob Chapman (@chapmanjacobd)
```

Comment:
Please move the `and` to Line 571

Reply:
Please format the last commit message as well.

Reply:
nobody else has an `and`. it likely makes more sense to remove that or place both names on one line. I'm not 100% sure what you want to do

either way I feel like that change is out of scope for this PR. I never modified line 570 or 571. if a previous PR was incorrect it is the maintainer's job to fix it. feel free to add commits to this branch

Reply:
I mean you should change it to:

```
trofi
danyspin97
jiegec and
chapmanjacobd
```

Reply:
mmmmk
I thought it was a line delimited list